src/box/lua/info.c

```diff
@@ -582,7 +582,7 @@ static int
 lbox_info_listen(struct lua_State *L)
 {
 	/* NULL is ok, no need to check. */
-	char addrbuf[SERVICE_NAME_MAXLEN];
+	char addrbuf[(SERVICE_NAME_MAXLEN + 2) * IPROTO_LISTEN_SOCKET_MAX];
```

Comment:
Why +2?

Reply:
in `iproto_bound_address` we add additional ", "  after each listen address

Reply:
Fixed